tools/picard.wdl

```diff
@@ -292,12 +299,19 @@ task merge_sam_files {
     String outfile_name = prefix + ".bam"
 
     command <<<
+        set -euo pipefail
+
         picard -Xmx~{java_heap_size}g MergeSamFiles \
             ~{sep(" ", input_arg)} \
             OUTPUT=~{outfile_name} \
+            --ASSUME_SORTED \
```

Comment:
I need to run it and see what errors. Picard's docs are a mess. I'm not bothering linking them because I don't particularly trust them.

But I'll mess around with it and make sure whatever gets merged runs